src/modules/configurator/components/saved-configurations/ConfigurationsSaved.tsx

```diff
@@ -0,0 +1,3 @@
+export default function ConfigurationsSaved() {
+  return <></>;
```

Comment:
Empty component? If you have something that is work in progress, you can add TODO comment, so a reviewer knows that it's WIP. And also so you can later find it easier

Reply:
Added a comment for work in progress.